frontend/src/components/application.js

```diff
@@ -3,10 +3,133 @@
  * @requires react-router-dom
  * @author Erik Hanstad
  */
-import React from 'react';
+import React, { useState, useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import 'bootstrap/dist/css/bootstrap.min.css';
+import { Button, Form, Card } from 'react-bootstrap';
+import DatePicker from 'react-datepicker';
+import 'react-datepicker/dist/react-datepicker.css';
+import { getCompetence, sendApplication } from '../redux/actions/applyActions';
 
 function Application() {
-  return <div>Application</div>;
+  const [yoe, setYoe] = useState('');
+  const [startDate, setStartDate] = useState(new Date());
+  const [endDate, setEndDate] = useState(new Date());
+  const [availability, setAvailability] = useState([]);
+  const [competences, setCompetence] = useState([]);
+  const apply = useSelector((state) => state.apply);
+  const user = useSelector((state) => state.auth.user);
+  const dispatch = useDispatch();
+
+  /*
+   * changes the state for year of experience
+   */
+  const sendApply = () => {
```

Comment:
från use case:

7. The system presents all the information the applicant has entered in
steps 1-6.
8. The applicant chooses to hand in the job application or to cancel.
9. The system registers the job application and displays a confirmation
message.

skulle kunna öppna en "modal" när man trycket på submit som visar all info man skrivit in sen får man trycka confirm/cancel